tests/compiler.js

```diff
@@ -475,6 +475,38 @@
             },
             'showing test\nshowing 1\nshowing 2\nshowing 3\n');
         });
+        /* global Set */
+        if (typeof Set === 'function') {
+          it('should work with Set bultin', function() {
```

Comment:
It looks like you have a typo here (and below) for 'builtin'.

Would it also be possible to rewrite this to use `this.skip`? e.g.:

```javascript
it('should work with the Set builtin', function() {
  /* global Set */
 if (typeof Set === 'undefined') {
    this.skip();
  } else {
    equal(/* ... */);
  }
});
```

Reply:
Done